content/src/fieldConfig/profile.json

```diff
@@ -101,13 +102,14 @@
       "taxPin": {
         "default": {
           "header": "Tax PIN",
-          "description": "What is your `Tax PIN|tax-pin`?",
+          "description": "What is your `Tax PIN|tax-pin` ?",
```

Comment:
It seems like there's a lot of lines of code changes that are adding a space in between the text and the question mark? This doesn't seem desirable - should be removed?

Reply:
Agreed, I will fix that, not sure how that got in there.